### PR TITLE
LP-2276 Fix all linting issues of homepage and student_dashboard

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2276](https://philanthropyu.atlassian.net/browse/LP-2276) Fix all linting issues of homepage and student_dashboard

### CODE REVIEW

File: lms/djangoapps/student_dashboard/views.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_enrolled_past_courses(request, course_enrollments):
             else:
                 enrolled.append(course)
 
-    for card, courses in past_course_cards.items():
+    for _, courses in past_course_cards.items():
```
Comment (asadali145) [inline lms/djangoapps/student_dashboard/views.py:40]:
```suggestion
    for courses in past_course_cards.values():
```

**Review by asadali145 — APPROVED** [review]
A small suggestion, Rest LGTM

**Comment by zahid-arbisoft** [comment]
> **pylint:** ❌
> **pycodestyle:** ✔️
> **iSort:** ✔️
> 
> **Please fix following pylint issues:**
> 
> ```
> ************* Module edx-platform.lms.djangoapps.homepage.custom_context_processors
> E: 4, 0: Unable to import 'edx_notifications.server.web.utils' (import-error)
> 
> -----------------------------------
> Your code has been rated at 9.44/10
> ```

@asadali145 I have updated the code. Please ignore this error, or it quality check fails. Actually, it has cached, the requirements and in that cache edx_notifications is not installed. When pushed a dummy test commit with changes in requirements, quality check passed, because it downloaded cache again. But now, I have reverted that commit. I think, old cache will be used

**Review by MunamMubashir-arbisoft — APPROVED** [review]
LGTM!

**Pull request merged**